src/Silex/Provider/TwigServiceProvider.php

```diff
@@ -54,15 +54,16 @@ public function register(Container $app)
         $app['twig.number_format.thousands_separator'] = ',';
 
         $app['twig'] = function ($app) {
-            $app['twig.options'] = array_replace(
+            $options = array_replace(
                 [
                     'charset' => $app['charset'],
                     'debug' => $app['debug'],
                     'strict_variables' => $app['debug'],
-                ], $app['twig.options']
+                ],
+                $app['twig.options']
```

Comment:
missing comma at the end

Reply:
I don't understand how that fixes the issue. Here, if the option is a closure, that will fail.

Reply:
Hmm, I don't actually understand how it would fail. I'm only reading a value from the container without overriding it anymore. It was failing before when it was a closure.

> missing comma at the end

This is a function call. Trailing commas in function calls are planned for PHP 7.3.
I've re-formatted it as per PSR-2 which states that when one argument is on its own line, every argument must be on its own line. An alternative formatting is just putting the array elements on their own line:

```php
$options = array_replace([
    'charset' => $app['charset'],
    'debug' => $app['debug'],
    'strict_variables' => $app['debug'],
],$app['twig.options']);
```